ci/e2e-compose.yml.base

```diff
@@ -0,0 +1,20 @@
+version: "3.7"
+
+services:
+  ajuna-node:
+    image: backend-devel.azurecr.io/node:@@HASH@@
```

Comment:
This is perfectly fine for me but, just to know it, could it be done using an environment variable instead of replacing the string?

Reply:
Hm, not really, since that section of the compose file cannot expand any variable passed to it, that's why we use the substitution.